velox/vector/FlatVector.h

```diff
@@ -202,7 +202,7 @@ class FlatVector final : public SimpleVector<T> {
   T* mutableRawValues() {
     if (!values_ || !values_->unique()) {
       BufferPtr newValues =
-          AlignedBuffer::allocate<T>(BaseVector::length_, values_->pool());
+          AlignedBuffer::allocate<T>(BaseVector::length_, BaseVector::pool());
```

Comment:
QQ : Why is this needed for the current change ?

Reply:
This is needed to avoid a crash when values_ is a nullptr.